test/validate/state_demand_test.py

```diff
@@ -2,37 +2,29 @@
 
 import logging
 
+import pyarrow.parquet as pq
 import pytest
 
-from pudl import validate as pv
+from pudl.workspace.setup import PudlPaths
 
 logger = logging.getLogger(__name__)
 
 
 @pytest.mark.parametrize(
-    "df_name,expected_rows",
-    [("predicted_state_hourly_demand", 6_706_318)],
+    "resource_id,expected_rows",
+    [("out_ferc714__hourly_estimated_state_demand", 6_706_318)],
```

Comment:
resource_id because its called a resource in our metadata? not asset_name or table_name?

Reply:
I used `resource_id` because that's what we call these strings in the context of our metadata classes, like `Package.from_resource_ids()`